### PR TITLE
Feature/ui improvements v2

### PR DESCRIPTION
**Pull request by henry54809**
I was able to cherry pick my change from the old branch.

### CODE REVIEW

File: daq/host.py [hunk 1]
```diff
@@ -595,20 +593,18 @@ def _docker_callback(self, return_code=None, exception=None):
...
         module_config_path = os.path.join(self._host_tmp_path(), self._MODULE_CONFIG)
+        remote_paths = {}
         for result_type, path in ((ResultType.REPORT_PATH, report_path),
                                   (ResultType.ACTIVATION_LOG_PATH, activation_log_path),
                                   (ResultType.MODULE_CONFIG_PATH, module_config_path)):
             if os.path.isfile(path):
                 self._report.accumulate(self.test_name, {result_type: path})
-                self._gcp.upload_file(path, self._get_unique_upload_path(path))
+                remote_paths[result_type.value] = self._upload_file(path)
+        self.record_result(self.test_name, state=state, code=return_code, exception=exception, \
```
Comment (grafnu) [inline daq/host.py:606]: no line continuation

File: firebase/public/index.html [hunk 2]
```diff
@@ -44,5 +44,7 @@ <h2>Users</h2>
   <div class="daq-text">Web version <span id="deploy-version"></span></div>
 </body>
 <script>
-  document.getElementById('deploy-version').innerHTML = daq_deploy_version;
+  if(typeof daq_deploy_version !== 'undefined'){
```
Comment (grafnu) [inline firebase/public/index.html:47]: should be a space after the if and before {

File: firebase/public/main.js [hunk 3]
```diff
@@ -110,7 +110,11 @@ function setGridValue(row, column, runid, value) {
         targetElement.setAttribute('runid', runid);
       }
       if (value) {
-        targetElement.innerHTML = value;
+        if(append){
```
Comment (grafnu) [inline firebase/public/main.js:113]: spaces

File: firebase/public/main.js [hunk 4]
```diff
@@ -199,6 +203,24 @@ function getResultStatus(result) {
   return '????';
 }
 
+function handleFileLinks(port, runid, result, type) {
+  const paths = Object.keys(result).filter(key => key.indexOf("path") >= 0);
+  if(paths.length){
```
Comment (grafnu) [inline firebase/public/main.js:208]: spaces

File: firebase/public/main.js [hunk 5]
```diff
@@ -199,6 +203,24 @@ function getResultStatus(result) {
...
+    let col = result.name;
+    if (result.name == "terminate") {
+      col = "report";
+    }
+    paths.forEach((path) => {
+      let name = path.replace("_path", "");
+      if(type == "port"){
```
Comment (grafnu) [inline firebase/public/main.js:215]: spaces

File: firebase/public/main.js [hunk 6]
```diff
@@ -264,10 +284,13 @@ function makeActivateLink(element, port) {
   element.onclick = () => activateRun(port)
 }
 
-function addReportBucket(origin, row, runid, reportName) {
+function addReportBucket(row, col, runid, path, name) {
   const storage = firebase.app().storage();
-  storage.ref().child(reportName).getDownloadURL().then((url) => {
-    setGridValue(row, 'report', runid, `<a href="${url}">${reportName}</a>`);
+  if(!name) {
```
Comment (grafnu) [inline firebase/public/main.js:289]: space

File: firebase/public/main.js [hunk 7]
```diff
@@ -418,7 +439,7 @@ function triggerSite(db, site_name) {
     watcherAdd(ref, "runid", latest, (ref, runid_id) => {
       console.log('runid', runid_id);
       watcherAdd(ref, "test", undefined, (ref, test_id) => {
-	console.log('test', test_id);
+	      console.log('test', test_id);
```
Comment (grafnu) [inline firebase/public/main.js:442]: indentation

File: daq/host.py (hunk 1 above)
Comment (henry54809) [reply daq/host.py:606]: I can't figure out how to make everything fit under 100 characters without making the code convoluted. It's 12 characters over the limit.

**Comment by grafnu** [comment]
but you don't need a line continuation (trailing \) if you have a trailing
comma inside of ()'s

**Comment by henry54809** [comment]
PTAL

**Pull request merged**